internal/store/demand_source_account.go

```diff
@@ -0,0 +1,42 @@
+package store
+
+import (
+	"database/sql"
+
+	"github.com/bidon-io/bidon-backend/internal/admin"
+)
+
+type DemandSourceAccountRepo = resourceRepo[admin.DemandSourceAccount, admin.DemandSourceAccountAttrs, demandSourceAccount, *demandSourceAccount]
+
+type demandSourceAccount struct {
+	Model
+	DemandSourceID int64          `gorm:"column:demand_source_id;type:bigint;not null"`
+	UserID         int64          `gorm:"column:user_id;type:bigint;not null"`
+	Type           string         `gorm:"column:type;type:varchar;not null"`
+	Extra          map[string]any `gorm:"column:extra;type:jsonb;default:'{}';serializer:json"`
+	IsBidding      *bool          `gorm:"column:bidding;type:boolean;default:false"`
```

Comment:
Do we need ptr if we set `not null` for bool column?

Reply:
Yes, GORM [does not update zero values from struct](https://gorm.io/docs/update.html#Updates-multiple-columns). So updating from `true` to `false` would not be possible without a pointer.